tests/unit/cluster/atomic-slot-migration.tcl

```diff
@@ -290,7 +290,7 @@ start_cluster 3 3 {tags {external:skip cluster} overrides {cluster-node-timeout
 
         # verify data
         assert_morethan [R 0 dbsize] 0
-        assert_equal [R 0 dbsize] [R 1 dbsize]
+        # assert_equal [R 0 dbsize] [R 1 dbsize]
```

Comment:
is this causing test failure? maybe we can use wait_for_ofs_sync?

Reply:
ah okay, they are both master and now as we filter unowned keys from dbsize, this test fails there.